### PR TITLE
WIP: Fix  snake code

### PR DESCRIPTION
**Pull request by prashant-subedi**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 \.ipynb_checkpoints
-*.ipynb
\ No newline at end of file
+*.ipynb
+**/venv/
```
Comment (Rrp13) [inline .gitignore]: **? kina deko directly /folder garda hunna or **/venv/ covers it automatically

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,4 @@
...
+.idea/
```
Comment (Rrp13) [inline .gitignore]: what is .idea?

File: Snake Game/main.py [hunk 3]
```diff
@@ -2,187 +2,135 @@
 import math
 import random
 
+STARTING_SPEED = 10
+ACCELARATION = 1
+# You don't really have to use this image though
+# Can just draw a rect
```
Comment (Rrp13) [inline Snake Game/main.py]: i wanted to tara tye documentation ramro nabhetera ho

File: Snake Game/main.py [hunk 4]
```diff
@@ -2,187 +2,135 @@
...
+    if snake_head[1] > obstacle[1] + ALL_OBJECTS_WIDTH:
+        return False
+    return True
 
-    
-        
-#Game loop
+
+def move_snake(snake, direction, elongate):
+    new_snake = [move_snake_head(snake[0], direction)] + snake[:-1]
```
Comment (Rrp13) [inline Snake Game/main.py]: yo line chai bujina

Comment (Rrp13) [reply Snake Game/main.py]: buje maile bujna ta
tara yedi coordinate ko spacing width bhanda badi vayo vane gap dekhina thalxa ni ta

Comment (Rrp13) [reply Snake Game/main.py]: i like that technique very much tara. buje paxi ta twaa being haha

File: .gitignore (hunk 1 above)
Comment (prashant-subedi) [reply .gitignore]: Repo has multiple projects. This says to ignore venv everywhere.

File: .gitignore (hunk 2 above)
Comment (prashant-subedi) [reply .gitignore]: Pycharm thing

File: Snake Game/main.py (hunk 3 above)
Comment (prashant-subedi) [reply Snake Game/main.py]: :D

File: Snake Game/main.py (hunk 4 above)
Comment (prashant-subedi) [reply Snake Game/main.py]: Yes...But it still has bugs. :D

**Pull request closed without merging**